src/vde_backend/sync_flukso.py

```diff
@@ -337,21 +336,21 @@ def generateHome(tmpo_session, hid, home_sensors, since_timing, to_timing):
 		sensors.append(Sensor(tmpo_session, row["flukso_id"], sid, since_timing, to_timing))
 
 	home = Home(home_sensors, since_timing, to_timing, hid, sensors)
-	home.showQueryInfo()
+	home.show_query_info()
 
 	return home
 
 
-def testSession(sensors_config):
+def test_session(sensors_config):
 	""" 
 	test each sensor and see if tmpo accepts or refuses the sensor when
 	syncing
 	"""
 	path = TMPO_FILE
 	if not path:
-		path = getProgDir()
+		path = get_prog_dir()
 
-	for sid, row in sensors_config.getSensorsConfig().iterrows():
+	for sid, row in sensors_config.get_nb_homes().iterrows():
```

Comment:
Is it still right to iterrows on a number ? You did the same modification over all the file